wgpu-core/src/command/clear.rs

```diff
@@ -66,6 +60,12 @@ whereas subesource range specified start {subresource_base_array_layer} and coun
         subresource_base_array_layer: u32,
         subresource_array_layer_count: Option<NonZeroU32>,
     },
+    #[error("failed to create view for clearing texture {texture:?} at mip level {mip_level}, layer {layer}")]
+    FailedToCreateTextureViewForClear {
```

Comment:
what are the possible reasons we could fail to create this view?

Reply:
whatever reason there was, it's gone now. Removing this value again